src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabitManager.java

```diff
@@ -0,0 +1,82 @@
+package seedu.duke.atomichabit.atomichabit;
+import java.util.HashMap;
+import java.util.Scanner;
+
+import seedu.duke.atomichabit.command.*;
+import seedu.duke.command.BadCommandException;
+import seedu.duke.command.CommandParser;
+public class AtomicHabitManager {
+    private final CommandParser parser = new CommandParser();
+    private final String HABIT_DIVIDER = "—---------------------------------------------------------------";
+    private AtomicHabitList habitList;
+    private static String userInput;
+
+    private static Scanner myScanner = new Scanner(System.in);
+
+    /**
+     * Method to be called by MainManager when user wishes to utilise atomichabits
+     */
+    public void run() {
+        start();
+        runCommands();
+    }
+
+    /**
+     * Requests user for input
+     * @return user input in a String
+     */
+    public static String takeInput() {
+        String line = myScanner.nextLine();
+        return line;
```

Comment:
Either return myScanner.nextLine() directly, or use a more descriptive name for `line` e.g. `nextCommand`. 

Be weary of `NoSuchElementException` as well, add a `try...catch` block around `Scanner.nextline()`